packages/tracing/src/Tracer.ts

```diff
@@ -140,12 +140,12 @@ class Tracer implements TracerInterface {
     * @param error - Error to serialize as metadata
     */
   public addErrorAsMetadata(error: Error): void {
-    if (this.tracingEnabled === false) {
+    if (!this.tracingEnabled) {
```

Comment:
Nice, we have another PR (#442) that is moving all these `this.tracingEnabled` to use `this.isTracingEnabled()`. Depending on which one gets merged first we might have to solve some conflicts but no big deal.

Reply:
Sure.

Reply:
@dreamorosi ok i have merged in the charges